mcc_generated_files/tmr0.c

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
As discussed, this file should be deleted. I can deal with it if mcc gives too much trouble.

Reply:
I figured it out, I didn't realize I needed to clean and build after deleting it